### ISSUE TITLE
Hide images from screenreader on search results

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1083]
When a user uses a screenreader, Images should be hidden on search results.

**Comment by jasoncorum** [comment]
@caseyedavis12 @afred: I was just doing a little online research to get a better sense of how this could be done and it looks like there is a lot of discussion on whether or not content and images should be adjusted specifically for screenreaders. Is this a direct recommendation? 

If so, it looks like it is difficult to reliably accomplish: http://stackoverflow.com/questions/7712167/is-it-possible-to-use-javascript-to-detect-if-a-screen-reader-is-running-on-a-us

It general it looks like recommendations are are just to make sure there is alt text for images.

**Comment by jasoncorum** [comment]
@caseyedavis12 @afred: So I did a little extra poking around and it looks like we can just use aria-hidden="true" for images we don't want read by a screenreader: https://www.w3.org/TR/wai-aria/states_and_properties#aria-hidden
